exports.go

```diff
@@ -2,33 +2,42 @@ package actor
 
 import (
 	"reflect"
+	"strings"
 	"unsafe"
 )
 
+func fail(errorMessage string) bool {
+	guestError(stringToPointer(errorMessage), uint32(len(errorMessage)))
+	return false
+}
+
 //go:export __guest_call
 func guestCall(operationSize uint32, payloadSize uint32) bool {
 	operation := make([]byte, operationSize) // alloc
 	payload := make([]byte, payloadSize)     // alloc
 	guestRequest(bytesToPointer(operation), bytesToPointer(payload))
 
-	if f, ok := allHandlers[string(operation)]; ok {
-		response, err := f(payload)
-		if err != nil {
-			message := err.Error()
-			guestError(stringToPointer(message), uint32(len(message)))
-
-			return false
+	op := string(operation)
```

Comment:
The original design for this (as inferred by the README) is that dispatch happens by the full operation name. Operation name _must be opaque_ to wasmCloud. This means we can't split on commas or periods or assume that there is a service name prefix on the operation name. 

The goal for the code generation was to generate strongly-typed wrapper functions that would register specific handlers, e.g. `RegisterHttpRequestHandler(handler)`